components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -0,0 +1,87 @@
+feature, and you should not either.
+{{< /warning >}}
+
+### System requirements
+Currently, Habitat Builder requires Chef Automate to be installed on the same host for
+authentication purposes. Outside the boundaries of a proof-of-concept exercise, we
+recommend against running installations of Chef Automate and Habitat Builder on
+the same host.
+
+#### Hardware
+The minimum with which Chef Automate and Habitat Builder can be deployed on a single host
+is:
+
+* 16 GB of RAM.
+* 130 GB of disk space, available to /hab.
+* 4 vCPUs.
+* Inbound network connectivity from LAN (HTTP/HTTPS) is required for internal clients to access the on-premises Chef Habitat Builder.
+
+For deployments that are expected to see production-scale workload, we recommend:
+* 64 GB of RAM.
+* 200 GB of diskspace, available to /hab.
+* 16 vCPUs.
+
+Roughly 80 GB of the disk space is designated for Chef Automate; the rest is used for
+Habitat Builder and the artifacts it stores. The current implementation uses Minio for
+Habitat artifact storage; we do not support using Artifactory for artifact storage.
+
+#### Operating system
+Chef Automate and Habitat Builder require
+
+* a Linux kernel of version 3.2 or greater
+* systemd as the init system
+* useradd
+* curl or wget
+* The shell that starts Automate should have a max open files setting of at least 65535
+
+### Deploy on-premises Habitat Builder
+Deploying Habitat Builder with Chef Automate requires a Chef Automate license. If you
+already have a Chef Automate license, you may use it for the deployment. Otherwise, you
+will have the opportunity to download a trial license when you sign in to Chef Automate
+after going through the deployment process.
+
+If you are deploying Habitat Builder with Chef Automate in an airgapped environment,
+follow <relref "airgapped-installation.md">the documentation on building an airgap bundle</relref>.
+
+To deploy Chef Automate and Habitat Builder, specify both the `builder` and `automate`
+products on the command line. For example,
+
+```shell
+sudo chef-automate deploy --product builder --product automate
+```
+
+### Sign in to Habitat Builder
+1. Sign in to Chef Automate using the <relref "install.md#open-chef-automate">credentials provided during deployment.</relfef>
+1. Go to https://automate.example.com/bldr.
+1. Select the Sign In button in the upper right corner.
+1. Select Sign In With Chef Automate.
+1. Use the same credentials as in Step 1 to sign in to Habitat Builder.
+
+See the <ref "https://www.habitat.sh/docs/using-builder/">Habitat Builder documentation</ref> for setting up a Builder access token, creating a Builder origin, etc.
+
+The Automate-deployed Habitat Builder currently supports authentication with local users only.
+
+### Backup Habitat Builder
+Builder backups are performed as part of the <relref "backup.md">Chef Automate backup process</relref>. The current implementation supports filesystem backups only.
```

Comment:
i'm not sure this is true. I think S3 backups should just work

Reply:
@jaym Does #2393 look reasonable? If so, I will take out the part about filesystem backups only.